app/src/main/java/com/example/konrad/decentwork/Login.java

```diff
@@ -0,0 +1,130 @@
+package com.example.konrad.decentwork;
+
+import android.app.ProgressDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Window;
+import android.widget.Button;
+import android.widget.EditText;
+import android.content.Intent;
+import android.view.View;
+import android.widget.Toast;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Login extends AppCompatActivity {
+
+    private static final String TAG = "Login";
+    private static final String URL_FOR_LOGIN = "https://XXX.XXX.X.XX/login.php";
```

Comment:
I think it's better if we make one class to make connection with API and then create constans like BASE_URL or something instead of writing full url to every class.